From ab33369c7a1c4638901a7c3705d03bce173c648d Mon Sep 17 00:00:00 2001
From: chandavath <rcrajagopal@habana.ai>
Date: Mon, 12 Feb 2024 10:32:36 +0200
Subject: [PATCH] [SW-137121] gaudi3: erom pcie: cleaned up unused pcie function

As part of gaudi3, this function is not required/used.
As part of gaudi2, this function is already part of common file.

Change-Id: I51f4c20f959fdd4b8efb28dce31267f48beccd74
Signed-off-by: chandavath <rcrajagopal@habana.ai>
---

diff --git a/app/ext_rom/erom_pcie.c b/app/ext_rom/erom_pcie.c
index d40ec84..3b373cc 100644
--- a/app/ext_rom/erom_pcie.c
+++ b/app/ext_rom/erom_pcie.c
@@ -16,13 +16,8 @@
 #include <uart.h>
 #include <common.h>
 #include <boot_if.h>
-
-#include <psoc_global_conf_regs.h>
-#include <pcie_dbi_regs.h>
-#include <pcie_phy_regs.h>
 #include <arcboot_export.h>
 #include <gaudi2/gaudi2_security.h>
-#include <gaudi2/gaudi2_bootrom.h>
 
 #define ARCBOOT_PRIMARY			0
 #define ARCBOOT_SECONDARY		1
@@ -47,23 +42,6 @@
 __weak void init_prstn(void) {}
 __weak void set_pci_msix_pba_offset_val(void) {}
 __weak void ppboot_log_waiting_for_pcie(void) {}
-__weak void preboot_init_pcie_phy(void)
-{
-	/* Reset PCIe PHY */
-	regmap_writel(0, mmPSOC_GLOBAL_CONF_PCIE_PHY_RST_N);
-
-	udelay(100);
-
-	regmap_writel(1, mmPCIE_PHY_PHYX_SRAM_BYPASS_0);
-	regmap_writel(1, mmPCIE_PHY_PHYX_SRAM_BYPASS_1);
-	regmap_writel(1, mmPCIE_PHY_PHYX_SRAM_BYPASS_2);
-	regmap_writel(1, mmPCIE_PHY_PHYX_SRAM_BYPASS_3);
-
-	udelay(10);
-
-	/* Take the PHY out of reset */
-	regmap_writel(1, mmPSOC_GLOBAL_CONF_PCIE_PHY_RST_N);
-}
 
 /* TODO: move this function to device independent common file  */
 static bool hl_pcie_link_up(void)
